pinot-controller/src/main/java/com/linkedin/pinot/controller/ControllerConf.java

```diff
@@ -80,6 +80,9 @@
   private static final long DEFAULT_SEGMENT_UPLOAD_TIMEOUT_IN_MILLIS = 600_000L; // 10 minutes
   private static final int DEFAULT_REALTIME_SEGMENT_METADATA_COMMIT_NUMLOCKS = 64;
 
+  private static final String ENABLE_STORAGE_QUOTA_CHECK = "controller.enable.storage.quota.check";
+  private static final boolean DEFAULT_STORAGE_QUOTA_CHECK = false;
```

Comment:
If the storage quota was enabled by default before this change, I think we should turn this on by default. Otherwise, it's not backward compatible and may affect to users outside.

Reply:
Updated the default.